controllers/chores.js

```diff
@@ -1,24 +1,133 @@
 const express = require('express');
 const router = express.Router();
+const db = require('../models');
+const logger = require('../logger');
 
 router.get('/get-chores', (req, res) => {
-    res.send('/get-chores working');
+    db.chore.findAll()
+    .then(data => {
+        res.status(200);
+        return res.json(data);
+    })
+    .catch(err => {
+        logger.error(err);
+        res.status(500);
+        return res.json({ errMessage: 'Server Error' });
+    });
 });
 
 router.get('/get-chore-by-id', (req, res) => {
-    res.send('/get-chore-by-id working');
+    const choreId = req.body.id;
+
+    db.chore.findAll({
+        where: {
+            id: choreId
+        }
+    })
+    .then(data => {
+        res.status(200);
+        return res.json(data);
+    })
+    .catch(err => {
+        logger.error(err);
+        res.status(500);
+        return res.json({ errMessage: 'Server Error' });
+    });
 });
 
 router.post('/add-chore', (req, res) => {
-    res.send('/add-chore working');
+    const { 
+        choreName, 
+        status, 
+        frequencyId,
+        categoryId, 
+        assigneeId, 
+        difficulty, 
+        notes, 
+        createdAt, 
+        updatedAt } = req.body;
+    //Will this be calculated on backend and saved when 'complete' box checked?
```

Comment:
That's correct.  Also, createdAt and updatedAt don't need to be passed in from the user.  Since this route creates a new chore, you can use the current date and time for both values

Reply:
removed createdAt & updatedAt